io.github.getnf.embellish.yml

```diff
@@ -0,0 +1,37 @@
+id: io.github.getnf.embellish
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: embellish
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --filesystem=xdg-data/embellish:create
```

Comment:
```suggestion
```

This won't be needed. Flatpak sets it to `~/.var/app/$FLATPAK_ID/data`

Reply:
This should be XDG base directory compliant https://github.com/getnf/embellish/blob/bbec1d3f4796c162b9901577a7a45351971a4d63/internal/types/types.go#L114

You are supposed to access this path relative to XDG_DATA_HOME not HOME.

Reply:
Embellish also has a CLI/TUI binary available on github, and I am thinking it would be nice if both the GUI and TUI/CLI builds have access to the same db.

If you still don't think it is necessary, I will change it.

Reply:
Package the CLI in the Flatpak too? You can run it with `flatpak run --command=cli-command <appid>`

Reply:
But then each tool has its own data home and won't be able too see the same db file.

Reply:
I mean in this flatpak. Then they will share the same `XDG_DATA_HOME` which is `~/.var/app/$FLATPAK_ID/data`